### PR TITLE
Platform.io Bidder Adapter update.  Prebid v1.0.

### PR DESCRIPTION
**Pull request by varashellov**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other

## Description of change

Platform.io Bidder Adapter update.  Prebid v1.0.
Adapter does not support external JS library

- test parameters for validating bids
```
{
  bidder: 'platformio',
  params: {
    pubId: '28082',
    siteId: '26047',
    placementId: '123',
    size: '250X250'
  }
}
```

- sk@ultralab.by
- [x] official adapter submission

### CODE REVIEW

File: test/spec/modules/platformioBidAdapter_spec.js [hunk 1]
```diff
@@ -1,156 +1,96 @@
...
-  afterEach(function () {
-    stubLoadScript.restore();
-  });
-
-  describe('creation of bid url', function () {
-    if (typeof ($$PREBID_GLOBAL$$._bidsReceived) === 'undefined') {
-      $$PREBID_GLOBAL$$._bidsReceived = [];
+import {expect} from 'chai';
+import {spec} from 'modules/platformioBidAdapter';
+import bidManager from 'src/bidmanager';
```
Comment (matthewlane) [inline test/spec/modules/platformioBidAdapter_spec.js]: `bidManager` isn't used in the test, line can be removed

File: test/spec/modules/platformioBidAdapter_spec.js [hunk 2]
```diff
@@ -1,156 +1,96 @@
...
+import {getTopWindowLocation} from 'src/utils';
+import {newBidder} from 'src/adapters/bidderFactory';
```
Comment (matthewlane) [inline test/spec/modules/platformioBidAdapter_spec.js]: `newBidder` isn't used in the test, line can be removed

File: modules/platformioBidAdapter.js [hunk 3]
```diff
@@ -1,63 +1,125 @@
-var bidfactory = require('src/bidfactory.js');
-var bidmanager = require('src/bidmanager.js');
-var adloader = require('src/adloader.js');
-var utils = require('src/utils.js');
-var CONSTANTS = require('src/constants.json');
-var adaptermanager = require('src/adaptermanager');
```
Comment (matthewlane) [inline modules/platformioBidAdapter.js]: Preferable to start files with content rather than a blank line, line can be removed

**Comment by varashellov** [comment]
@matthewlane Hey, thanks for the feedback. Everything was updated, please verify. Looking forward to you reply.

**Pull request merged**